ansible/playbooks/roles/repository/files/download-requirements/requirements/x86_64/images.yml

```diff
@@ -2,7 +2,7 @@
 images:
   haproxy:
     'haproxy:2.2.2-alpine':
-      sha1: dff8993b065b7f7846adb553548bcdcfcd1b6e8e
+      sha1: 41839b8a10cdb17e3e322e8be8bb0ba4004ef09e
 
   image-registry:
     'registry:2.8.0':
```

Comment:
I guess sha1 for `registry:2.8.0` image is also different now

Reply:
You are right, I mislooked this one (due to the allow_mismatch flag ;) )